schemas/panels/time-series/time-series.cue

```diff
@@ -31,8 +31,8 @@ import (
 }
 
 #palette: {
-	mode: "auto" | "categorical"
-	// colors: [...string]; // TODO: add colors to override ECharts theme
+	mode:               "auto" | "categorical"
+	singleSeriesColor?: string
```

Comment:
@Nexucis I’m reading #557 again, sounds like this should be an option on the query instead of how I have it here?

If so, what level would it be, on TimeSeriesQuery or nested inside PrometheusTimeSeriesQuery?

Reply:
Hello @sjcobb we were just discussing about that with Antoine and saw your comment, so let me jump on it :D

We were talking about a use case that seems to be exactly that: 
In our tool we embed perses panel displaying ALERTS query result. For now we display currently only firing result as we don't have a way to visually separate them. 
Having the possibility to make two queries with let's say red for firing and gray for pending for example would be very interesting for us.

Reply:
Yeahh like right after I completed this approach, I read Augustin’s ticket again and it finally clicked why on the query would be so much better 😂 

We have a metadata option on TimeSeriesQuery already, I’d propose we add a `config` since styles like color aren’t really metadata. Then I can probably reuse some of this work and make it happen

There are some other updates in this branch though like the useMemo changes so I’ll break those out into a separate PR